specs/GetFavicon.md

```diff
@@ -0,0 +1,149 @@
+# Background
+A Favicon is an asset which is a part of a webpage, and typically displayed on each tab. Developers would
```

Comment:
This sentence leaves out that its a small icon and describing it as part of a webpage is sort of misleading since its a separate file. MDN describes it as
```suggestion
A favicon (favorite icon) is a tiny icon included along with a website, which is displayed in places like the browser's address bar, page tabs and bookmarks menu. Developers would
```